src/nyc_trees/nyc_trees/context_processors.py

```diff
@@ -10,23 +10,12 @@
 
 from django.utils.timezone import make_aware, utc
 
-from apps.survey.models import Blockface, Survey
+from apps.survey.models import (Blockface, Survey, Territory,
+                                BlockfaceReservation)
+from apps.users.models import TrustedMapper
 
 
 def settings(request):
```

Comment:
I'm down for renaming it.  How does `config_js` sound, to match the HTML partial?